test-network-function/accesscontrol/suite.go

```diff
@@ -46,132 +44,104 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 		ginkgo.BeforeEach(func() {
 			common.ReloadConfiguration(&configData)
 		})
-
+		common.ReloadConfiguration(&configData)
 		testNamespace(&configData)
 
 		testRoles(&configData)
 
-		// Former "container" tests
 		defer ginkgo.GinkgoRecover()
 
 		// Run the tests that interact with the pods
 		ginkgo.When("under test", func() {
-			conf := configpkg.GetConfigInstance()
-			podsUnderTest := conf.PodsUnderTest
-			gomega.Expect(podsUnderTest).ToNot(gomega.BeNil())
-			for _, pod := range podsUnderTest {
-				var podFact = testcases.PodFact{Namespace: pod.Namespace, Name: pod.Name, ContainerCount: 0, HasClusterRole: false, Exists: true}
-				// Gather facts for containers
-				podFacts, err := testcases.LoadCnfTestCaseSpecs(testcases.GatherFacts)
+			podUnderTest := configData.PodsUnderTest
+			gomega.Expect(podUnderTest).ToNot(gomega.BeNil())
```

Comment:
this is exactly the same thing that's causing beforeEach error in CI.
you should not be testing something outside of IT block, there's no guarantee this podUnderTest is not nil

Reply:
fixed